src/style.css

```diff
@@ -0,0 +1,58 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars:not(:first-child) {
+  margin-top: -6px;
+}
+
+.stars--star {
+  width: 20px;
+  height: 20px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  margin-left: 2px;
+  margin-top: 2px;
+}
+
+.stars--1 > .stars--star:first-child {
+  width: 20px;
+  height: 20px;
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+}
```

Comment:
I think only `background-image` changes
so no need to duplicate all the other rules